src/processTargets/targets/ScopeTypeTarget.ts

```diff
@@ -90,7 +90,7 @@ export default class ScopeTypeTarget extends BaseTarget {
 
   getRemovalRange(): Range {
     return this.removalRange_ != null
-      ? getTokenRemovalRange(this, this.removalRange_)
+      ? this.removalRange_
```

Comment:
This was trying to cleanup whitespace on removal range passed in, but it wasn't doing it correctly, and we don't do that when they pass in delimiters, so it seemed inconsistent.  We can revisit in the future if we want to clean up whitespace on removal ranges.  As mentioned above, nothing failed when I changed this

Reply:
This is a regression. Granted in some weird edge cases. Unfortunately we were lacking tests.
in markdown for example
`"# Name "`
`chuck name` will on main remove that trailing white space. Your changes will leave the white space.

Reply:
As discussed at meet-up, let's keep this removed for now and handle it as part of #855